src/EoS/Nuclear/cubinterp_some.cpp

```diff
@@ -62,34 +65,20 @@ void nuc_eos_C_cubinterp_some( const real x, const real y, const real z,
    real dx, dy, dz, dxi, dyi, dzi;
    real delx, dely, delz;
    real u[4], v[4], w[4];
-   real r[4], q[4];
-   real vox;
-   int  ix, iy, iz;
-   int  nxy;
-
-   nxy = nx*ny;
+   real r, q, vox;
+   int  ix, iy, iz, iv;
+   int  nxy  = nx*ny;
+   int  nxyz = nxy*nz;
```

Comment:
`const int nxy` and `const int nxyz`. Add `const` for other variables whenever appropriate (check other files too).